test/commonControlTests/ListControl.spec.ts

```diff
@@ -29,68 +29,90 @@ suite('ListControl e2e tests', () => {
         createControlTree(state: any): Control {
             return new ListControl({
                 id: 'apple',
-                validation: (state, input) => ['iPhone', 'iPad', 'MacBook'].includes(state.value!) ? true : { renderedReason: 'Apple Suite category validation failed' },
+                validation: (state, input) =>
+                    ['iPhone', 'iPad', 'MacBook'].includes(state.value!)
+                        ? true
+                        : { renderedReason: 'Apple Suite category validation failed' },
                 listItemIDs: ['iPhone', 'iPad', 'MacBook'],
                 slotType: 'AppleSuite',
                 confirmationRequired: true,
                 prompts: {
                     valueSet: '',
-                }
+                },
             });
         }
     }
 
     test('product value valid, needs explicit affirming', async () => {
         const requestHandler = new ControlHandler(new ListControlManager());
         await testE2E(requestHandler, [
-            'U: iPhone', TestInput.of(SingleValueControlIntent.of('AppleSuite', { AppleSuite: 'iPhone' })),
+            'U: iPhone',
+            TestInput.of(SingleValueControlIntent.of('AppleSuite', { AppleSuite: 'iPhone' })),
             'A: Was that iPhone?',
-            'U: Yeah.', TestInput.of(IntentBuilder.of(AmazonIntent.YesIntent)),
-            'A: Great.'
+            'U: Yeah.',
+            TestInput.of(IntentBuilder.of(AmazonIntent.YesIntent)),
+            'A: Great.',
         ]);
     });
 
     test('product value after disaffirmation, requires request value act', async () => {
         const requestHandler = new ControlHandler(new ListControlManager());
         await testE2E(requestHandler, [
-            'U: iPhone', TestInput.of(SingleValueControlIntent.of('AppleSuite', { AppleSuite: 'iPhone' })),
+            'U: iPhone',
+            TestInput.of(SingleValueControlIntent.of('AppleSuite', { AppleSuite: 'iPhone' })),
             'A: Was that iPhone?',
-            'U: No.', TestInput.of(IntentBuilder.of(AmazonIntent.NoIntent)),
-            'A: My mistake. What is your selection? Some suggestions are iPhone, iPad or MacBook.'
+            'U: No.',
+            TestInput.of(IntentBuilder.of(AmazonIntent.NoIntent)),
+            'A: My mistake. What is your selection? Some suggestions are iPhone, iPad or MacBook.',
         ]);
     });
 
     test('product value set and changing it requires confirmation and value changed act', async () => {
         const requestHandler = new ControlHandler(new ListControlManager());
         await testE2E(requestHandler, [
-            'U: iPhone', TestInput.of(SingleValueControlIntent.of('AppleSuite', { AppleSuite: 'iPhone' })),
+            'U: iPhone',
+            TestInput.of(SingleValueControlIntent.of('AppleSuite', { AppleSuite: 'iPhone' })),
```

Comment:
nit: Is this ok? Kinda looks weird where we had individual lines alternative for User and Alexa earlier?

Reply:
Yes there are going to be some cases where the preferred format isn't achieved.  I see many reports of people eventually saying they don't mind the change and that the opinionated formatter is liberating.  I'm a little skeptical, but I'm willing to try.  We might find different ways to address it, and worst case we can use 
```
// prettier-ignore
```